src/sagas.js

```diff
@@ -23,7 +23,41 @@ import {
 } from 'ovirt-ui-components'
 
 // import store from './store'
-import { persistState, getSingleVm } from './actions'
+import {
+  persistState,
```

Comment:
I would vote for moving all *VmDialog related sagas to a separate file. This is starting to become unmanageable.